lib/Map/RegionProvider.js

```diff
@@ -116,6 +117,17 @@ var RegionProvider = function(regionType, properties) {
      * @type {Object}
      */
     this._idIndex = {};
+
+    /**
+     * The URL of a pre-generated JSON file containing just a long list of IDs for a given
+     * layer attribute, in the order of ascending feature IDs (fids). If defined, it will
+     * be used in preference to requesting those attributes from the WFS server.
+     * @type {String}
+     */
+    this.regionIdList = properties.regionIdList;
```

Comment:
A nit-picky suggestion - it's potentially confusing to have something whose name ends in `List` actually be a URL (ie a `String`) - could you change this to `this.regionIdListURL` or something similar?